src/errors.rs

```diff
@@ -180,6 +180,7 @@ pub fn permission_denied() -> DenoError {
   )
 }
 
+#[derive(Debug)]
```

Comment:
Is this necessary, given that you also implement an explicit formatter?

Reply:
Yes without it I get this error because I've implemented `fmt::Display` but not `fmt::Debug`
```
error[E0599]: no method named `expect` found for type `std::result::Result<(), errors::RustOrJsError>` in the current scope
   --> ../../src/isolate.rs:787:10
    |
787 |         .expect("execute_mod error");
    |          ^^^^^^
    |
    = note: the method `expect` exists but the following trait bounds were not satisfied:
            `errors::RustOrJsError : std::fmt::Debug`
```